### PR TITLE
revise GETTING_STARTED.md

### PR DESCRIPTION
**Pull request by chenyang23333**
Thank you for your contribution to the MindYOLO repo.
Before submitting this PR, please make sure:

- [ ] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindyolo/blob/master/CONTRIBUTING.md)
- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

## Motivation
revise GETTING_STARTED.md

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: GETTING_STARTED.md [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
+2. Download the corresponding pre-train checkpoint from the [model zoo](MODEL_ZOO.md) of each model.
+3. We provide `demo/predict.py` that can demo built-in configs. Run it with:
 
 ```
-python demo/predict.py --config ./configs/yolov7/yolov7.yaml --device_target=Ascend --weight=MODEL.WEIGHTS --image_path /PATH/TO/IMAGE.jpg
+# Run with Ascend (By default)
+python demo/predict.py --config ./configs/yolov7/yolov7.yaml --weight=/path_to_ckpt/WEIGHT.ckpt --image_path /path_to_image/IMAGE.jpg
+
+# Run with GPU
+python demo/predict.py --config ./configs/yolov7/yolov7.yaml --device_target=GPU --weight=/path_to_ckpt/WEIGHT.ckpt --image_path /path_to_image/IMAGE.jpg
```
Comment (zhanghuiyao) [inline GETTING_STARTED.md:18]: device_target参数是不是可以放到最后

File: GETTING_STARTED.md [hunk 2]
```diff
@@ -46,28 +48,24 @@ to understand its behavior. Some common arguments are:
   ```
   </details>
 
-* To train a model:
+* To train a model on 8 NPUs:
```
Comment (zhanghuiyao) [inline GETTING_STARTED.md:51]: NPUs -> NPUs/GPUs?

File: GETTING_STARTED.md [hunk 3]
```diff
@@ -46,28 +48,24 @@ to understand its behavior. Some common arguments are:
...
-  mpirun --allow-run-as-root -n 8 python train.py
-  e.g.:
-  mpirun --allow-run-as-root -n 8 python train.py --config ./configs/yolov7/yolov7.yaml --device_target Ascend --is_parallel True > log.txt 2>&1 &
-  tail -f ./log.txt
+  mpirun --allow-run-as-root -n 8 python train.py --config ./configs/yolov7/yolov7.yaml  --is_parallel True
   ```
 
-* The configs are made for 8 NPU/GPU training.
-To train on 1 NPU/GPU, you may need to change some parameters, e.g.:
+* To train a model on 1 NPU:
```
Comment (zhanghuiyao) [inline GETTING_STARTED.md:56]: 1. NPU -> NPU/GPU/CPU
2. Notes: (1) The default hyper-parameter are used for 8 card training, and some parameters may need to be adjusted in the case of a single card. (2) The default device is Ascend, and other devices can be modified by 'device_target' specifies that Ascend/GPU/CPU is currently supported.

File: GETTING_STARTED.md (hunk 1 above)
Comment (chenyang23333) [reply GETTING_STARTED.md]: 已修改

File: GETTING_STARTED.md (hunk 2 above)
Comment (chenyang23333) [reply GETTING_STARTED.md]: 已修改

File: GETTING_STARTED.md (hunk 3 above)
Comment (chenyang23333) [reply GETTING_STARTED.md]: 已修改

**Review by zhanghuiyao — DISMISSED** [review]

File: GETTING_STARTED.md [hunk 4]
```diff
@@ -1,21 +1,23 @@
...
 ### Inference Demo with Pre-trained Models
 
-1. Pick a model and its config file from
+1. Pick a model and its config file from the
   [model zoo](MODEL_ZOO.md),
-  for example, `./configs/yolov7/yolov7.yaml`.
-2. Download the corresponding pretrain checkpoint from the link in the [model zoo](MODEL_ZOO.md) of each model.
-3. We provide `demo/predict.py` that is able to demo builtin configs. Run it with:
+  such as, `./configs/yolov7/yolov7.yaml`.
+2. Download the corresponding pre-train checkpoint from the [model zoo](MODEL_ZOO.md) of each model.
```
Comment (CaitinZhao) [inline GETTING_STARTED.md:10]: pretrained

File: GETTING_STARTED.md [hunk 5]
```diff
@@ -24,7 +26,7 @@ to understand its behavior. Some common arguments are:
 
 ### Training & Evaluation in Command Line
 
-* Prepare your dataset in YOLO format. If train with COCO dataset, prepare it from yolov5 or darknet.
+* Prepare your dataset in YOLO format. If trained with COCO (YOLO format), prepare it from yolov5 or the darknet.
```
Comment (CaitinZhao) [inline GETTING_STARTED.md:29]: 加个链接？

File: GETTING_STARTED.md [hunk 6]
```diff
@@ -1,21 +1,23 @@
 ## Getting Started with MindYOLO
 
-This document provides a brief intro of the usage of builtin command-line tools in MindYOLO.
+This document provides a brief intro of the usage of built-in command-line tools in MindYOLO.
```
Comment (SamitHuang) [inline GETTING_STARTED.md:3]: intro of  -> introduction to

File: GETTING_STARTED.md [hunk 7]
```diff
@@ -1,21 +1,23 @@
...
 
-1. Pick a model and its config file from
+1. Pick a model and its config file from the
   [model zoo](MODEL_ZOO.md),
-  for example, `./configs/yolov7/yolov7.yaml`.
-2. Download the corresponding pretrain checkpoint from the link in the [model zoo](MODEL_ZOO.md) of each model.
-3. We provide `demo/predict.py` that is able to demo builtin configs. Run it with:
+  such as, `./configs/yolov7/yolov7.yaml`.
+2. Download the corresponding pre-train checkpoint from the [model zoo](MODEL_ZOO.md) of each model.
+3. We provide `demo/predict.py` that can demo built-in configs. Run it with:
```
Comment (SamitHuang) [inline GETTING_STARTED.md:11]: To run YOLO object detection with the built-in configs, please run:

File: GETTING_STARTED.md [hunk 8]
```diff
@@ -1,21 +1,23 @@
...
+
+# Run with GPU
+python demo/predict.py --config ./configs/yolov7/yolov7.yaml --weight=/path_to_ckpt/WEIGHT.ckpt --image_path /path_to_image/IMAGE.jpg --device_target=GPU
 ```
 
-The configs are made for training, therefore we need to specify `MODEL.WEIGHTS` to a model from model zoo for inference.
-This command will run the inference and show visualizations in an OpenCV window.
 
 For details of the command line arguments, see `demo/predict.py -h` or look at its source code
 to understand its behavior. Some common arguments are:
```
Comment (SamitHuang) [inline GETTING_STARTED.md:23]: its source code 给个链接到config.py代码文件。
its behavior -> their behavior

File: GETTING_STARTED.md [hunk 9]
```diff
@@ -46,28 +48,24 @@ to understand its behavior. Some common arguments are:
   ```
   </details>
 
-* To train a model:
+* To train a model on 8 NPUs/GPUs:
```
Comment (SamitHuang) [inline GETTING_STARTED.md:51]: 8 -> multiple

File: GETTING_STARTED.md (hunk 4 above)
Comment (chenyang23333) [reply GETTING_STARTED.md]: 已修改

File: GETTING_STARTED.md (hunk 5 above)
Comment (chenyang23333) [reply GETTING_STARTED.md]: 已修改

File: GETTING_STARTED.md (hunk 6 above)
Comment (chenyang23333) [reply GETTING_STARTED.md]: 已修改

File: GETTING_STARTED.md (hunk 7 above)
Comment (chenyang23333) [reply GETTING_STARTED.md]: 已修改

File: GETTING_STARTED.md (hunk 8 above)
Comment (chenyang23333) [reply GETTING_STARTED.md]: 已修改

File: GETTING_STARTED.md [hunk 10]
```diff
@@ -1,30 +1,32 @@
...
+# Run with GPU
+python demo/predict.py --config ./configs/yolov7/yolov7.yaml --weight=/path_to_ckpt/WEIGHT.ckpt --image_path /path_to_image/IMAGE.jpg --device_target=GPU
 ```
 
-The configs are made for training, therefore we need to specify `MODEL.WEIGHTS` to a model from model zoo for inference.
-This command will run the inference and show visualizations in an OpenCV window.
 
-For details of the command line arguments, see `demo/predict.py -h` or look at its source code
-to understand its behavior. Some common arguments are:
+For details of the command line arguments, see `demo/predict.py -h` or look at its [source code](https://github.com/mindspore-lab/mindyolo/tree/master/configs)
```
Comment (zhanghuiyao) [inline GETTING_STARTED.md:22]: 链接不对，放predict.py的链接

Comment (chenyang23333) [reply GETTING_STARTED.md]: 已修改

**Review by zhanghuiyao — APPROVED** [review]

**Review by zhanghuiyao — APPROVED** [review]

**Review by CaitinZhao — APPROVED** [review]

**Pull request merged**